cypress/integration/pages/frontPage/tests.js

```diff
@@ -134,6 +134,27 @@ export const testsThatFollowSmokeTestConfig = ({ service, pageType }) =>
             },
           );
         });
+        it('should contain Useful Links if usefulLinks block data exists', () => {
+          cy.request(`${config[service].pageTypes.frontPage.path}.json`).then(
+            ({ body }) => {
+              const pageData = body.content.groups;
+              const usefulLinks = pageData.find(data => {
+                return data.type === 'useful-links';
+              });
+              const hasStrapline = usefulLinks && 'strapline' in usefulLinks;
```

Comment:
We are checking two things here, not just that there is a strapline, so I would rename `hasStrapline` to something like `isValidUsefulLinks`